rules/ruleset.go

```diff
@@ -574,7 +574,7 @@ func NewRuleSetFromSchema(version int, rs *schema.RuleSet, opts Options) (RuleSe
 func NewEmptyRuleSet(namespaceName string, meta UpdateMetadata) MutableRuleSet {
 	rs := &ruleSet{
 		uuid:         uuid.NewUUID().String(),
-		version:      1,
+		version:      0,
```

Comment:
nit: can use `kv. UninitializedVersion` or simply omit this since it's initialized to 0 by default anyway?